modules/utility/init.zsh

```diff
@@ -22,8 +22,12 @@ if [[ ! -z ${terminfo[colors]} ]] && (( ${terminfo[colors]} >= 8 )); then
     # BSD
 
     # colours for ls and completion
-    export LSCOLORS='exfxcxdxbxGxDxabagacad'
-    export LS_COLORS='di=34:ln=35:so=32:pi=33:ex=31:bd=36;01:cd=33;01:su=31;40;07:sg=36;40;07:tw=32;40;07:ow=33;40;07:'
+    if [[ -z ${LSCOLORS} ]]; then
```

Comment:
I'd like to see this style of these checks replaced with `(( ${+LSCOLORS} ))`. If everyone was using zsh 5.3+, we could use the nice `[[ -v LSCOLORS ]]` syntax, but alas.